README.md

```diff
@@ -82,6 +82,7 @@
 - [Firefox](https://www.mozilla.org/en-US/firefox/new/)
 - [Ungoogled Chromium](https://ungoogled-software.github.io/)
 - [Tor Browser](https://www.torproject.org/)
+- [LibreWolf](https://librewolf-community.gitlab.io/)
```

Comment:
Can you change it to a nested element below Firefox and mention it is a privacy-focused fork? Just like this:

* Firefox
    * LibreWolf - Privacy-focused Firefox fork.